### PR TITLE
Homework 2

### PR DESCRIPTION
**Pull request by HayrapetyanD**

### CODE REVIEW

File: homework_2/js/index.ts [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+  tagName: string;
+  attributes: AttributesType;
+  children: ChildrenType;
+
+  constructor(
+    tagName: string,
+    attributes: AttributesType,
+    children: ChildrenType
+  ) {
+    this.tagName = tagName;
```
Comment (aonarek) [inline homework_2/js/index.ts:20]: The super method should be called here

File: homework_2/js/index.ts [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+}
+
+function el(
+  tagName: string,
+  attributes: AttributesType,
+  children: ChildrenType
+): DomElement {
+  const className =
+    tagName.charAt(0).toUpperCase() + tagName.slice(1) + "Element";
+
```
Comment (aonarek) [inline homework_2/js/index.ts:65]: It was necessary to try to implement multiple classes of DomElement.

Comment (hayrapetyandavit) [reply homework_2/js/index.ts:65]: this is done to reduce the code, in fact the classes are implemented

Comment (aonarek) [reply homework_2/js/index.ts:65]: but in this case, I have the option to make a tag with any name, which should not be allowed.

for example. el("test", { style: "font-size: 14px" }, "Test Content")

Comment (hayrapetyandavit) [reply homework_2/js/index.ts:65]: if we have a tag name validation, will it be correct?

**Pull request merged**